doc/source/overview/algorithms.md

```diff
@@ -45,13 +45,13 @@ minimally and necessarily absent to maintain the original prediction (a PN acts
+**Kernel Shapley Additive Explanations (Kernel SHAP)**: attribute the change of a model output with respect to a given baseline (e.g., average over a reference set) to each of the input features. This is achieved for each feature in turn, by averaging the difference in the model output observed when the feature whose contribution is to be estimated is part of a group of "present" input features and the value observed when the feature is excluded from said group. The features that are not "present" (i.e., are missing) are replaced with values from a background dataset. This algorithm can be used to explain regression models. [Documentation](../methods/KernelSHAP.ipynb), [continuous data](../examples/kernel_shap_wine_intro.ipynb), [more continuous data](../examples/kernel_shap_wine_lr.ipynb), [categorical data](../examples/kernel_shap_adult_lr.ipynb).
 
-**Tree Shapley Additive Explanation (Tree SHAP)**: attribute the change of a model output with respect to a baseline (e.g., average over a reference set or inferred from node data) to each of the input features. Similar to Kernel SHAP, the shap value of each feature is computed by averaging the difference of the model output observed when the feature is part of a group of "present" features and when the feature is excluded from said group, over all possible subsets of "present" features. Different estimation procedures for the effect of selecting different subsets of present features on the model output give rise to the interventional feature perturbation and the path-dependent feature perturbation variants of Tree SHAP. This algorithm can be used to explain regression models. [Documentation](../methods/TreeSHAP.ipynb), [interventional feature perturbation Tree SHAP](interventional_tree_shap_adult_xgb.ipynb), [path-dependent feature perturbation Tree SHAP](../examples/path_dependent_tree_shap_adult_xgb.ipynb).
+**Tree Shapley Additive Explanations (Tree SHAP)**: attribute the change of a model output with respect to a baseline (e.g., average over a reference set or inferred from node data) to each of the input features. Similar to Kernel SHAP, the shap value of each feature is computed by averaging the difference of the model output observed when the feature is part of a group of "present" features and when the feature is excluded from said group, over all possible subsets of "present" features. Different estimation procedures for the effect of selecting different subsets of present features on the model output give rise to the interventional feature perturbation and the path-dependent feature perturbation variants of Tree SHAP. This algorithm can be used to explain regression models. [Documentation](../methods/TreeSHAP.ipynb), [interventional feature perturbation Tree SHAP](../examples/interventional_tree_shap_adult_xgb.ipynb), [path-dependent feature perturbation Tree SHAP](../examples/path_dependent_tree_shap_adult_xgb.ipynb).
```

Comment:
Should we quote "present" everywhere (or nowehere)? I decided to quote as the reader might wonder what on earth we mean by a present feature?